### ISSUE TITLE
Function name should start with a lowercase letter for Composable Functions

### ISSUE THREAD

**Issue by AbdullahJaved-dev** [JLLeitschuh/ktlint-gradle#736]
`Function name should start with a lowercase letter (except factory methods) and use camel case (cannot be auto-corrected)`

I am trying to disable this warning but it's not working. Below is the code that I'm using. 

```
id("org.jlleitschuh.gradle.ktlint") version "12.0.3"

ktlint {
    version.set("1.1.0")
    debug.set(true)
    enableExperimentalRules.set(true)
    additionalEditorconfig.set(
        mapOf(
            "max_line_length" to "off",
            "ktlint_function_naming_ignore_when_annotated_with" to "Composable"
        ),
    )
}
```

**Comment by wakingrufus** [comment]
although rules can be disabled via additionalEditorconfig, not _every_ configuration works this way. try putting this configuration in `.editorconfig` and see if that works

**Comment by AbrahamCardenesAcid** [comment]
Hi, I'm having the same problem. Is there any solutions or workaround?

**Comment by AbrahamCardenesAcid** [comment]
My solution in: https://github.com/JLLeitschuh/ktlint-gradle/issues/742 
I hope it works for you too!

**Comment by wakingrufus** [comment]
#742 contains the answer
